### ISSUE TITLE
Make "image list" command reusable

### ISSUE THREAD

**Issue by ploober** [erichaagdev/KappaBot#24]
Right now the command `/kb image list` returns a hard-coded string with the URL https://kb.gorlah.com/. Clearly, this makes the command unusable for instances of KappaBot running anywhere else.

A configurable URL would probably be a good starting point.

Otherwise, if we can figure out some way to provide usable listing functionality without directing to an external website, that might be cool. I'm not sure if we have any viable options to accomplish this in a generic way though.

**Comment by erichaagdev** [comment]
Yeah this one is hard. Originally, when KappaBot was small, I just dumped every known image to the chat. Obviously, this very quickly became unreadable. I am almost inclined to fully remove the command for the time being. The `/kb image search` command is a much better alternative I think. 

I am also fairly confident that URL does not even work anymore. I wrote that site in PHP years ago so it definitely needs to be rewritten itself. This might be something interesting to create a "project" for include #25 with this?

**Comment by ploober** [comment]
The URL was not working earlier when I tried it.

**Comment by erichaagdev** [comment]
It's up now. 😄
